### ISSUE TITLE
What kind of encoding is this???

### ISSUE THREAD

**Issue by grepwood** [bazel-contrib/rules_oci#679]
https://github.com/bazel-contrib/rules_oci/blob/a32f3bfe82464cda39959a6ae24b8ec547c67abc/examples/setup_assertion_repos.bzl#L62

Why can't it be something familiar like base16 is used for every other sha256 in the project?

**Comment by thesayyn** [comment]
> https://github.com/bazel-contrib/rules_oci/blob/a32f3bfe82464cda39959a6ae24b8ec547c67abc/examples/setup_assertion_repos.bzl#L62

https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity

> Why can't it be something familiar like base16 is used for every other sha256 in the project?

It can be but i am not sure why that matters?

**Comment by grepwood** [comment]
> It can be

Great! Thank you!

> why that matters?

For 2 reasons.
1. This isn't a widely used format. A lot of people are going to be left scratching their heads wondering how can they reproduce this, should they ever want to confirm the checksum manually or write a patch that changes the behavior of this section of rules_oci. Your comment is already very helpful. Thank you very much!
2. The only other place I've seen this used was NixOS where the documentation would say nothing about what encoding is used. When I pried the community's minds for answers I've learned it's base32... but of what? Cabbage? Rubbish? Apparently so, because it's not the base32 of whatever package was being handled. So much for wild claims of reproducibility. With that experience, I've come to expect problems from this notation.

But since you described it so well, I consider this issue done and would like to thank you once more :)
